metashare/api/models.py

```diff
@@ -677,7 +680,18 @@ def finalize_provision(self, error=None):
     def queue_get_unsaved_changes(self):
         from .jobs import get_unsaved_changes_job
 
+        minutes_since_last_check = (
+            self.last_checked_unsaved_changes_at is not None
+            and timezone.now() - self.last_checked_unsaved_changes_at
+        )
+        should_bail = minutes_since_last_check and minutes_since_last_check < timedelta(
+            minutes=settings.ORG_RECHECK_MINUTES
+        )
+        if should_bail:
+            return
```

Comment:
@wlonk I think the front end should be able to bypass this check and force a refresh. Maybe using the `get_unsaved_changes` query param, or a new one.